dm-wolfmqtt-publisher-client-with-zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(MCXN947-mqtt-client-wolfMQTT)
```

Comment:
Should the project name include MCXN947 or maybe just have it match the `dm-wolfmqtt-publisher-client-with-zephyr`?